alacritty_terminal/src/config/window.rs

```diff
@@ -69,6 +69,24 @@ impl WindowConfig {
     }
 }
 
+impl Default for WindowConfig {
+    fn default() -> WindowConfig {
+        WindowConfig {
+            dimensions: Default::default(),
+            position: Default::default(),
+            padding: Default::default(),
+            decorations: Default::default(),
+            dynamic_padding: Default::default(),
+            startup_mode: Default::default(),
+            title: default_title(),
```

Comment:
Forgot to mention that in the last review, but would you mind moving this to the bottom? I think that would make it stand out a little more with all these `Default::default()`s. Top would work too I guess.

Reply:
Done.